internal/unionstore/memdb_test.go

```diff
@@ -928,6 +955,7 @@ func testMemDBCheckpoint(t *testing.T, buffer MemBuffer) {
 
 func TestBufferLimit(t *testing.T) {
 	testBufferLimit(t, newRbtDBWithContext())
+	testBufferLimit(t, newArtDBWithContext())
 }
 
 func testBufferLimit(t *testing.T, buffer MemBuffer) {
```

Comment:
Is all the previous memdb test executed for both `art` and `rbt` implementations by now?

Reply:
Snapshot read is not supported yet, all the other tests are running for both `art` and `rbt`.